dags/prio.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator, GKEClusterCreateOperator, GKEClusterDeleteOperator
+from utils.gke import create_gke_config
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.contrib.operators.s3_to_gcs_transfer_operator import S3ToGoogleCloudStorageTransferOperator # noqa
+
+DEFAULT_ARGS = {
+    'owner': 'hwoo@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2019, 5, 31),
+    'email': ['hwoo@mozilla.com', 'dataops+alerts@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=15),
+}
+
+cluster_name = 'gke-prio-a'
+
+gcp_conn_id = "google_cloud_prio_a"
+connection = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id)
+
+cluster_def = create_gke_config(
+    name=cluster_name,
+    service_account="prio-runner-a@moz-fx-priotest-project-a.iam.gserviceaccount.com",
+    owner_label="hwoo",
+    team_label="dataops"
+    )
+
+gke_dag = DAG(
+    'gke_prio_a',
+    default_args=DEFAULT_ARGS,
+    dagrun_timeout=timedelta(hours=6),
+    schedule_interval='0 2 * * *'
```

Comment:
I don't know what the scheduling is supposed to look like other than weekly.

Reply:
oh yeah, thanks